src/opa.js

```diff
@@ -240,6 +246,21 @@ class LoadedPolicy {
     this.wasmInstance.exports.opa_eval_ctx_set_input(ctxAddr, inputAddr);
     this.wasmInstance.exports.opa_eval_ctx_set_data(ctxAddr, this.dataAddr);
 
+
+    if(entrypoint) {
+      if(typeof entrypoint === 'number') {
+        this.wasmInstance.exports.opa_eval_ctx_set_entrypoint(ctxAddr, entrypoint);
+      } else if (typeof entrypoint === 'string') {
+        if(this.entrypoints.hasOwnProperty(entrypoint)) {
+          this.wasmInstance.exports.opa_eval_ctx_set_entrypoint(ctxAddr, this.entrypoints[entrypoint]);
+        } else {
+          throw `entrypoint ${entrypoint} is not valid in this instance!`;
+        }
```

Comment:
Support for _"passing a string"_ is nice. I wonder if we need the number case at all, then. but since it's straightforward, why not. 💭

Reply:
The main purpose of allowing entry by number was to avoid the string matching if someone has a fairly large number of entrypoints. But that lookup is pretty negligible so I'm guessing almost all usage will be with a string passed.